ref/general/csec-tls.adoc

```diff
@@ -0,0 +1,108 @@
+Therefore, client and server use these initial exchanges to generate a shared secret key that is used to symmetrically encrypt communications for the remainder of the session.
+Since this shared key is privately generated by each party, rather than being exchanged between them, there is less chance for key to be exposed to eavesdroppers.
+
+A basic TLS handshake comprises the following high-level steps:
+
+. The TLS client sends a `client hello` message, which includes the TLS version, a list of preferred CypherSuites and a random bite string that is used in subsequent computations.
+. The TLS server responds with a `server hello` message, which includes the CypherSuite, the session ID, another random byte string, and the server's digital certificate. The client can use this certificate to verify the server's authentication. If the server requires the client's certificate, it sends a client certificate request that lists the types of certificates it can accept.
+. The client verifies the server's digital certificate and sends an encrypted random byte string that is used to generate the shared secret key. If the server requested the client's certificate, the client sends either the certificate or a `no certificate alert`. This alert can cause the handshake to fail if the certificate is configured as mandatory by the server. If a certificate is sent, the server verifies the client's certificate.
+. The client sends a `finished` message, which is encrypted with the shared secret key and indicates that the client side of the handshake is complete.
+. The server sends a `finished` message, which is encrypted with the shared secret key and indicates that the server side of the handshake is complete.
+. For the remainder of the session, client and server can exchange symmetrically encrypted messages by using the shared secret key.
+
+=== The TLS record
+
+Once the TLS handshake establishes authentication and encryption details, the TLS record protocol handles the transmission and verification of encrypted messages during the session.
+The TLS record reads the messages to be transmitted, breaks them into manageable chunks and encrypts them according to the terms agreed on during the TLS handshake.
+The TLS record also specifies a means to compute the message digest so that message integrity can be verified by both parties.
+On the receiving side, the record decrypts the messages and verifies their integrity by comparing the message digests.
+
+== TLS and Open Liberty
+
+You can enable support for TLS in Open Liberty by adding the `transportSecurity-1.0` feature to your `server.xml` file.
+You can specify the details of your TLS settings by adding the  `ssl`, `sslDefault`, and `keyStore` elements to your `server.mxl` file.
+For more information, see the link:/docs/ref/feature/#transportSecurity.html[Transport Security feature].
+
+In addition to adding the `transportSecurity-1.0` feature, the `keystore` element must be added to your `server.xml` file.
```

Comment:
Same thing here as above. I _think_ that we talked about using the plain text name except in code examples?